mergify_engine/actions/__init__.py

```diff
@@ -63,6 +63,8 @@ class ActionFlag(enum.Flag):
     # Allow to return this command as pending if does not match its conditions
     # requirements.
     ALLOW_AS_PENDING_COMMAND = enum.auto()
+    # Once succeed the action must not be ran anymore
+    CAN_SUCCEED_ONLY_ONCE = enum.auto()
```

Comment:
I think the name is confusing
what about `SUCCESS_IS_FINAL_STATE`? or something like that.

Reply:
Or `NO_RUN_AFTER_SUCCESS`

Reply:
NO_RUN_AFTER_SUCCESS suggests we can still CANCEL, but we should not. SUCCESS_IS_FINAL_STATE looks good.